### DISCUSSION TITLE
7주차 과제

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#25] [Show and tell]
#과제 1
Data Visualization 아이디어 발전 + 스케치
+ 최소한 2개 이상의 화면 / 실제 데이터 3개 이상 디자인
+ 화면 1920x1080 사이즈에 맞춰 디자인
+ 블로그 글로

- - - - - - - -

#과제2
Dynamic Typography 아이디어 발전 + 스케치
+ 어떤 규칙에 의해 타입이 디자인 / 애니메이션 / 랜덤화 되는지 글로 설명
+ 5개 이상의 알파벳 디자인
+ 블로그 글로

- - - - - - - -

각 과제마다 블로그 글 하나씩 따로 만들어서 링크 2개 올려주세요!

과제 기한 - 월요일 밤 12시까지

이제부터는 과제전 + 기말과제 혼합이니, 매주 과제 퀄리티 / 기한 등 엄정히 평가하도록 하겠습니다-

**Comment by SeyounJung01** [comment]
정세윤

과제 1_https://blog.naver.com/seyounjj2001/222562431180
과제 2_https://blog.naver.com/seyounjj2001/222562439415

**Comment by KimHyunJuhg** [comment]
김현정

과제1 https://blog.naver.com/khj317888/222562402236
과제2https://blog.naver.com/khj317888/222562026780

**Comment by eunjinhong** [comment]
홍은진
과제1 https://blog.naver.com/tnekf1012/222562504807
과제2 https://blog.naver.com/tnekf1012/222562412544

**Comment by imyeeunok** [comment]
김예은
https://blog.naver.com/010312kye

**Comment by Joeun410** [comment]
김조은 
과제1 
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222562519656&categoryNo=14&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=11916
 과제2
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222562521386&categoryNo=15&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=3483

**Comment by jwejwepark** [comment]
박주혜

과제1
https://blog.naver.com/glqkfl0726/222562840707


과제2
https://blog.naver.com/glqkfl0726/222562523399
